client/my-sites/site-settings/form-general.jsx

```diff
@@ -133,11 +135,23 @@ class SiteSettingsFormGeneral extends Component {
 
 		if (
 			this.props.isSavingSettings &&
-			! nextProps.isSavingSettings &&
-			nextProps.isSaveRequestSuccessful
+			! nextProps.isSavingSettings
 		) {
-			nextProps.clearDirtyFields();
-			nextProps.markSaved();
+			if ( nextProps.isSaveRequestSuccessful ) {
+				nextProps.successNotice( nextProps.translate( 'Settings saved!' ), { id: 'site-settings-save' } );
+				nextProps.clearDirtyFields();
+				nextProps.markSaved();
+			} else {
+				let text;
+				switch ( nextProps.saveRequestError.error ) {
+					case 'invalid_ip':
+						text = nextProps.translate( 'One of your IP Addresses was invalid. Please try again.' );
```

Comment:
grok'ed it - and it looks like `invalid_ip` error code is part of the domains/DNS logic - I'll try to  test out in that area.